### ISSUE TITLE
Enhancement: I would like add some other payment and category

### ISSUE THREAD

**Issue by Arzzam** [gokulkrishh/expense.fyi#111]
If there is option to add Payment Method by the user on its own

Like I am using Splitwise as I am bachelor, my room mates will pay for me. so I need to note down as splitwise and here My room mates will be my payment mode and I can keep track of my friends who paid for me.

And Also When the user chose other category, we can give them another input field to write down the what catergory is this. 
Or like we can allow them to add categories as well.

If this is possible. I can also work on this onea as I am a developer and I would like to contribute to this project.

![image](https://github.com/gokulkrishh/expense.fyi/assets/92790024/af798426-91b8-4dd7-a026-40ee114b23b8)

**Comment by gokulkrishh** [comment]
Hi @Arzzam I had plan to do it myself but got caught up with other work. If you want to contribute you can, This feature could be good addition.

**Comment by Arzzam** [comment]
Yes @gokulkrishh , I would like to contribute but It might me helpful If there is any readme for setup the project in local

**Comment by Arzzam** [comment]
Hi @gokulkrishh,
Do you have any updates regarding setting up the project?
